desktop-app/app/components/ExtensionsManager/index.js

```diff
@@ -105,7 +105,8 @@ export default function ExtensionsManager({triggerNavigationReload}) {
   };
 
   const validateExtensionId = extensionId =>
-    extensionId.match(/^[a-z]+$/) && extensionId.length <= 32;
+    (extensionId.match(/^[a-z]+$/) && extensionId.length <= 32) ||
```

Comment:
@pajaydev should'n it be `extensionId.length === 32` ?

Reply:
Fixed it